examples/nominal.rs

```diff
@@ -0,0 +1,72 @@
+use actix_web::{App, web, Responder,
+  HttpResponse, get, test};
+use biscuit_auth::{Biscuit, KeyPair, 
+  macros::{biscuit, authorizer}, builder_ext::AuthorizerExt};
+use biscuit_actix_middleware::BiscuitMiddleware;
+
+#[get("/hello-admin")]
+async fn hello_admin(token: web::ReqData<Biscuit>) -> impl Responder {
+  // authorizer that allow only if role is admin
+  let mut authorizer = authorizer!(
+    r#"
+      allow if role("admin");
+    "#
+  );
+
+  // deny if no allow
+  authorizer.add_deny_all();
```

Comment:
nitpick: this is not necessary, no matching policies means that the request will be refused.

Reply:
fix in 415c84cef43bd9ab6dec0b60b82bfc9de34ced76